sdcm/cluster.py

```diff
@@ -5105,6 +5105,27 @@ def is_features_enabled_on_node(self, feature_list: list[str], node: BaseNode):
                 enabled_features_state.append(feature in enabled_features)
         return all(enabled_features_state)
 
+    def get_node_status_from_system_by(self, verification_node: BaseNode, *, ip_address: str = "", host_id: str = "") -> dict[str, Any]:
```

Comment:
Let's stop bloating `cluster.py`. This can also be moved under `raft` module and use `verification_node.parent_cluster.cql_connection_patient` 
Also add doclines explaining why it's needed and why current ways of getting node status is not good enough.

Reply:
Moved to raft.common as simple function